.github/workflows/sync-packgejson-and-changelog-after-auto-release.yml

```diff
@@ -0,0 +1,53 @@
+name: 🔁 Sync Package.json and CHANGELOG After Auto Release
```

Comment:
the name of this .yml file is way to long...and there is a typo 'packge'.

just 'after-release' and leave these desc as a comment in this file.

Reply:
changed to `after-auto-release`